conf/base.conf

```diff
@@ -124,6 +124,7 @@ puzzle {
     puzzle = puzzle2_puzzle
     round = puzzle2_round
     path = puzzle2_path
+    report = puzzle2_report
```

Comment:
This should not need a separate collection, but for now I did like that so if we scrape it, it's much easier to clean the db.